pages/cookie.tsx

```diff
@@ -5,9 +5,7 @@ export async function getStaticProps({ locale }: { locale: any }) {
   return {
     props: {
       metas: {
-        title: 'Cookie, Open Source Day 2023 - Florence',
-        description:
-          'Open Source Day 2023 coming on the 24th of March 2023. Stay tuned on our social'
+        title: 'Cookie, Open Source Day 2043 - Florence',
```

Comment:
```suggestion
        title: 'Cookie, Open Source Day 2024 - Florence',
```
2043 you are a clairvoyant

Reply:
Done, mr. president